tests/one_drive/test_one_drive.py

```diff
@@ -1,7 +1,11 @@
+import copy
```

Comment:
I think we should also add tests for elements of `parse_dates` that are not necessary datetime type !